### ISSUE TITLE
Altitude parsing in iGlide generated IGC file incorrect

### ISSUE THREAD

**Issue by tptolton** [skylines-project/skylines#408]
Altitude parsing is working incorrectly (https://skylines.aero/flights/53257/) with a file create in iGlide on iOS. File load to OLC worked without issues (http://www.onlinecontest.org/olc-2.0/gliding/flightinfo.html?flightId=1528570663).

.

**Comment by tptolton** [comment]
I had another problem with altitude parsing on another flight today. I noticed that iGlide now records the iPhone 6 pressure altitude value. I always forget to set the local altimeter setting (QNH) on the phone before flying, and the first few records before takeoff are recording a negative altitude value. After I changed the IGC file for those records by removing the "minus" (-) sign, the IGC file that was previously failing was imported successfully. That is probably the reason for the altitude parsing problems.
